exercises_on_code_quality/NumberFormat.js

```diff
@@ -0,0 +1,17 @@
+const numberFormat = function (number) {
```

Comment:
I would suggest to use arrow function or normal function syntax instead of assigning it to a variable